nvflare/fuel/utils/config.py

```diff
@@ -38,8 +38,10 @@ def config_ext_formats(cls):
 
 
 class Config(ABC):
```

Comment:
As our config/config service codes are growing larger, does it make sense to move it outside of `nvflare/fuel/utils`?